mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -241,8 +243,62 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
     }
   ]
 
+  // Asset sort bottom sheet list
+  const assetValue = 'Asset value'
+  const [assetSortType, setAssetSortType] = useState(assetValue) // to display selected sorted type text
+  const [isSorted, setIsSorted] = useState<boolean>(false) // to display acsending/descending icon
+  const [hideIcon, setHideIcon] = useState(false)
+  const [showAssetSortBottomSheet, setShowAssetSortBottomSheet] = useState(false)
+  const modifiedDenominationCurrency = useMemo(() => denominationCurrency === 'USDT' ? 'USD' : denominationCurrency, [denominationCurrency])
+  const sortTokensAssetOnType = useCallback((assetSortType: string): BalanceRowToken[] => {
+    // from assetSortList
+    switch (assetSortType) {
+      case (`Highest ${modifiedDenominationCurrency} value`):
+        return filteredTokens.sort((a, b) => {
+          return b.usdAmount.minus(a.usdAmount).toNumber()
+        })
+      case (`Lowest ${modifiedDenominationCurrency} value`):
+        return filteredTokens.sort((a, b) => {
+          return a.usdAmount.minus(b.usdAmount).toNumber()
+        })
+      case ('Highest token amount'):
+        return filteredTokens.sort((a, b) => {
+          return new BigNumber(b.amount).minus(new BigNumber(a.amount)).toNumber()
+        })
+      case ('Lowest token amount'):
+        return filteredTokens.sort((a, b) => {
+          return new BigNumber(a.amount).minus(new BigNumber(b.amount)).toNumber()
+        })
+      case ('A to Z'):
+        return filteredTokens.sort((a, b) => {
+          return a.displaySymbol.localeCompare(b.displaySymbol)
+        })
+      case ('Z to A'):
+        return filteredTokens.sort((a, b) => {
+          return b.displaySymbol.localeCompare(a.displaySymbol)
+        })
+      default:
+        return filteredTokens
```

Comment:
We should have a default sorting

Reply:
Updated! The default sorting is changed to `Highest{denomination} value`. Same with what we currently have in the app